### PR TITLE
Added dotmatrix function... so you can experience the 90s all over again

### PR DESCRIPTION
**Pull request by quephird**
This feature does not include sound effects though. :3

### CODE REVIEW

File: xenoglossia/builtins.py [hunk 1]
```diff
@@ -214,6 +214,18 @@ def buttify(input, *args):
     """
     return re.sub("(?i)(butt|but)", u"\U0001F351", input)
 
+_DOTMATRIX_CHARS = ["⡮⢵", "⣟⣳", "⢎⣉", "⣏⡱", "⣟⣋", "⡯⠍", "⢎⣥", "⡗⢺", "⡇", "⢉⠏", "⡧⢎", "⣇⣀", "⡗⢺"
```
Comment (mistydemeo) [inline xenoglossia/builtins.py]: Do you mind switching these to Unicode literals? For example, `u"⡮⢵"` That way you won't have to decode them from UTF-8 in the function, and it should work as-is on Python 3 too!

Comment (quephird) [inline xenoglossia/builtins.py]: I'm sorry!!! I got lazy this time around. :S

Comment (mistydemeo) [inline xenoglossia/builtins.py]: No worries! The `u` prefix with the unicode characters inside the literal would have worked too, btw.

Comment (quephird) [inline xenoglossia/builtins.py]: DOH... I'm suuuuuch a Python n00b

**Comment by mistydemeo** [comment]
Merged, thanks :D

**Pull request closed without merging**